lib/codec/rle.ts

```diff
@@ -42,7 +43,7 @@ exports.encodeValues = function(type, values, opts) {
     case 'BOOLEAN':
     case 'INT32':
     case 'INT64':
-      values = values.map((x) => parseInt(x, 10));
+      values = values.map((x) => parseInt(x as unknown as string, 10)); // We should fix unknown as string
```

Comment:
Implemented helper function to fix. Not sure about the Int64 but I do know that the toPrimitives that I worked on return `number` if the value given is number but `bigint` otherwise if thats what you were referencing